docs/sct-runners.md

```diff
@@ -22,3 +22,25 @@ hydra --execute-on-runner `cat sct_runner_ip` [Any hydra commands]
 hydra is syncing the whole SCT directory with rsync,
 try to make sure there's no huge files in SCT directory,
 because that would slow it down
+
+
+### process of updating sct runner images
+
+1. update code in sct_runner.py, give it a new version number
+2. Build all images
+    ```bash
+
+     ./sct.py create-runner-image -c aws -r eu-west-2 -z a
+
+    ./sct.py create-runner-image -c gce -r us-east1 -z a
+    SCT_GCE_PROJECT=gcp ./sct.py create-runner-image -c gce -r us-east1 -z a
+    SCT_GCE_PROJECT=gcp-local-ssd-latency ./sct.py create-runner-image -c gce -r us-east1 -z a
+
+    ./sct.py create-runner-image -c azure -r eastus -z a
+    ```
+3. update version on `aws_builder.py` and `gce_builder.py`
+4. build jenkins configuration with new sct runner image
+    ```bash
+    ./sct.py configure-jenkins-builders -c gce
+    ./sct.py configure-jenkins-builders -c aws
+    ```
```

Comment:
one more thing to update labels in jenkinsfiles and `vars/getJenkinsLabels.groovy`

Reply:
yeah, that's also part of game